Rakefile

```diff
@@ -0,0 +1,128 @@
+# frozen_string_literal: true
+
+require 'date'
+require 'json'
+require 'net/http'
+require 'optparse'
+
+require 'curb' # Curl Ruby, an external libcurl binding library
+
+UNDESIRED_LABELS = [
+  'refactoring', 'needs conceptual review', 'needs rebase', 'waiting for author'
+]
+GITHUB_API_URL = 'https://api.github.com/repos/bitcoin/bitcoin/pulls'
+HTTP_SUCCESS = 200
+HTTP_ERRORS = [
+  JSON::ParserError,
+  EOFError,
+  Errno::ECONNRESET,
+  Errno::EINVAL,
+  Net::HTTPBadResponse,
+  Net::HTTPHeaderSyntaxError,
+  Net::ProtocolError,
+  Timeout::Error
+]
+
+desc 'Create a new post file'
+namespace :posts do
+  task :new do
+    # Fetch user command line arguments.
+    pr, host, date = get_cli_options
+
+    # Set default values for host and date if not supplied by user.
+    host ||= 'jnewbery'
```

Comment:
I suggest leaving this as blank if not provided. The only person that this default could possibly benefit would be me, so it doesn't seem worthwhile adding it.

Reply:
Done.